R/detect_mistakes.R

```diff
@@ -203,9 +204,30 @@ detect_mistakes <- function(user,
       )
     }
 
+    
+    if (length(well_matched > 0)){
+      matched_user_names <- sort(rlang::names2(well_matched))
+      
+    if ( !allow_partial_matching ){
+      
+      matches <- which(sapply(lapply(remaining_solution_names,startsWith, matched_user_names),sum)==1)
+      matched_solution_name <- sort(remaining_solution_names[matches])
+      
+      return( 
+        pmatches_argument_name(
+          this_call = user,
+          this = as.character(user[matched_user_names]),
+          this_name = matched_user_names,
+          correct_name = matched_solution_name,
+          enclosing_call = enclosing_call,
+          enclosing_arg = enclosing_arg
+        )
+      )
+      
+    }
+    
+    
     # Remove partially matched arguments from further consideration
```

Comment:
Please align the white space

Reply:
I made some modifications, but not sure to understand "align the white space" :)